components/NextMeetup/style.scss

```diff
@@ -94,4 +95,97 @@
   .level > img {
     margin: 10px 3px 70px 0px;
   }
+
+  .signButton,
+  .arrow {
+    display: none;
+  }
+}
+
+@media screen and (min-width: 800px) {
+  .next {
+    display: inline-flex;
+    flex-wrap: wrap;
+    justify-content: space-around;
+    .arrow {
+      display: flex;
+      order: 0;
+      width: 100%;
```

Comment:
Wydaje mi się, że to sprawia, ze na moim kompie strzałka jest wielgaśna. Chyba nie potrzebujemy jej tak stylować, ona ma swój rozmiar.